### PR TITLE
Revert "Update Wrangler command docs"

### PR DESCRIPTION
**Pull request by deadlypants1973**
Reverts cloudflare/cloudflare-docs#4627

@petebacondarwin 

Pages `create` and `publish` are described as the same command. What is the difference between these two?

### CODE REVIEW

File: content/workers/wrangler/commands.md [hunk 1]
```diff
@@ -835,144 +785,6 @@ After starting `wrangler tail`, you will receive a live feed of console and exce
...
-{{<definitions>}}
-
-- `--project-name` {{<type>}}string{{</type>}}
-  - The name of the project you would like to list deployments for.
-
-{{</definitions>}}
-
-#### `create`
-
-Publish a directory of static assets as a Pages deployment.
```
Comment (deadlypants1973) [inline content/workers/wrangler/commands.md:926]: @petebacondarwin the description for `create` matches that of `publish`. What is the difference?

Comment (sidharthachatterjee) [reply content/workers/wrangler/commands.md:926]: `create` and `publish` are aliases of each other.

Comment (petebacondarwin) [reply content/workers/wrangler/commands.md:926]: Thanks for the info @sidharthachatterjee - should we only have one in the docs? And if so, which one?

**Comment by petebacondarwin** [comment]
@deadlypants1973 - what is the process here? Do we merge this and then I resubmit the original PR with amendments? Or do we make the amendments in this PR so that it only reverts the bits we are not happy with?

**Review by kodster28 — APPROVED** [review]

File: content/workers/wrangler/commands.md (hunk 1 above)
Comment (sidharthachatterjee) [reply content/workers/wrangler/commands.md:926]: I think having `publish` first class is a good idea. 

`wrangler pages publish <dir>`

Comment (petebacondarwin) [reply content/workers/wrangler/commands.md:926]: OK thanks.

**Pull request merged**